database/migration/src/main/scala/cromwell/database/migration/custom/BatchedTaskChange.scala

```diff
@@ -0,0 +1,140 @@
+package cromwell.database.migration.custom
```

Comment:
ToL: I'm open to other names for this package. I stopped myself from throwing it in yet-another generic `*.util` package.